test/common/helpers/api.js

```diff
@@ -82,7 +82,7 @@ var http = {
 function paramsHelper(url, params) {
 	if (
 		typeof params !== 'undefined' &&
```

Comment:
There is no difference between `typeof params !== 'undefined'` and `params !== undefined`